lib/models/asset-size-printer.js

```diff
@@ -5,46 +5,48 @@ const path = require('path');
 const walkSync = require('walk-sync');
 const workerpool = require('workerpool');
 
-module.exports = class AssetPrinterSize {
+module.exports = class AssetSizePrinter {
```

Comment:
We use `AssetSizePrinter` everywhere else, and the filename is `asset-size-printer` as well.